<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PenTools</class>
 <widget class="QWidget" name="PenTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>122</width>
    <height>482</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>122</width>
    <height>482</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Kalem Araçları</string>
  </property>
  <widget class="QPushButton" name="pen_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Kalem</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eraser_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Silgi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="color_red">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>130</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>QPushButton { background-color: red; border-radius: 5px; border: 1px solid gray; }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="color_blue">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>QPushButton { background-color: blue; border-radius: 5px; border: 1px solid gray; }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="color_black">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>130</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>QPushButton { background-color: black; border-radius: 5px; border: 1px solid gray; }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="color_green">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>165</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>QPushButton { background-color: green; border-radius: 5px; border: 1px solid gray; }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="color_custom1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>165</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>QPushButton { background-color: lightgray; border-radius: 5px; border: 1px solid gray; }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="color_custom2">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>165</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>QPushButton { background-color: darkgray; border-radius: 5px; border: 1px solid gray; }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="selected_color_indicator">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>40</width>
     <height>10</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>10</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>10</height>
    </size>
   </property>
   <property name="styleSheet">
    <string>background-color: red; border-radius: 5px; border: 1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_all_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>her şeyi sil</string>
   </property>
  </widget>
  <widget class="QPushButton" name="line_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Çizgi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rect_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Kare</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ellipse_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Daire</string>
   </property>
  </widget>
  <widget class="QPushButton" name="highlight_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Highlighter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Kapat</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>